cylc/flow/data_store_mgr.py

```diff
@@ -1233,6 +1233,7 @@ def insert_job(self, name, point_string, status, job_conf):
         Args:
             name (str): Corresponding task name.
             point_string (str): Cycle point string
+            status: ??
```

Comment:
Not really me - it's been added to the list of args at somepoint without documentation.

Reply:
It's the job state i.e. `running`, `submitted`, `failed`, `submit-failed`.

Reply:
```suggestion
status: Job-state - i.e. running, submitted, failed, submit failed
```